backend/src/utils/route-security.ts

```diff
@@ -63,17 +63,17 @@ const requestSecurityGuardNotebook = async (
 const requestSecurityGuard = async (
   fastify: KubeFastifyInstance,
   request: OauthFastifyRequest,
-  name: string,
   namespace: string,
   needsAdmin: boolean,
+  name?: string,
 ): Promise<void> => {
   const { notebookNamespace, dashboardNamespace } = getNamespaces(fastify);
   const username = await getUserName(fastify, request);
   const translatedUsername = usernameTranslate(username);
   const isReadRequest = request.method.toLowerCase() === 'get';
 
   // Check to see if a request was made against one of our namespaces
-  if (![notebookNamespace, dashboardNamespace, MODEL_REGISTRY_NAMESPACE].includes(namespace)) {
+  if (![notebookNamespace, dashboardNamespace].includes(namespace)) {
```

Comment:
Why is `MODEL_REGISTRY_NAMESPACE` removed?

Reply:
I have no idea how that happened